docs/src/content/docs/fr/guides/site-search.mdx

```diff
@@ -110,13 +112,62 @@ Avec cette configuration mise à jour, la barre de recherche de votre site ouvri
 
 #### Configuration de DocSearch
 
-Le module d’extension DocSearch de Starlight supporte également la personnalisation du composant DocSearch avec les options supplémentaires suivantes :
+Le module d’extension DocSearch de Starlight supporte la personnalisation du composant DocSearch avec les options supplémentaires suivantes spécifiées dans la configuration du module d’extension :
```

Comment:
We don't really have a real translation for "inline" and "en ligne" feels weird: https://french.stackexchange.com/questions/54639/what-is-the-translation-of-inline-in-french